tools/zapdir/main.go

```diff
@@ -0,0 +1,63 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/Microsoft/hcsshim"
```

Comment:
Use the internal package. The root package is deprecated.

Reply:
As a bonus, using the internal API is cleaner too.